neurodamus/node.py

```diff
@@ -702,22 +697,6 @@ def enable_stimulus(self):
             logging.info(" * [STIM] %s: %s (%s) -> %s", name, stim_name, stim_pattern, target_spec)
             self._stim_manager.interpret(target_spec, stim)
 
-    # -
-    def _enable_electrodes(self):
-        if SimConfig.use_coreneuron:
-            # Coreneuron doesnt support electrodes
-            return False
-        electrode_path = self._run_conf.get("ElectrodesPath")
-        if electrode_path is not None:
-            logging.info("ElectrodeManager using electrodes from %s", electrode_path)
-        else:
-            logging.info("No electrodes path. Extracellular class of stimuli will be unavailable")
-
-        self._elec_manager = Nd.ElectrodeManager(
-            electrode_path and Nd.String(electrode_path),
-            SimConfig.get_blueconfig_hoc_section("parsedElectrodes")
```

Comment:
Shall we also remove  the function definition of `get_blueconfig_hoc_section` ? I check it is only used for "parsedElectrodes".

Reply:
You are right. Removing!